install_libraries.txt

```diff
@@ -0,0 +1,4 @@
+pytest == 7.0.1
+flask == 2.0.3
+flask_restful == 0.3.9
+psycopg2 == 2.9.3
```

Comment:
the convention I've seen is to call this file `requirements.txt` - can you change it pls?